resources/views/tags/transfer.blade.php

```diff
@@ -17,12 +17,17 @@
             <p class="help-block mb-5">
                 {{ __('tags.transfer.description') }}
             </p>
-
             <div class="field-campaign mb-5">
                 <label>{{ __('tags.fields.tag') }}</label>
-                {!! Form::select('tag', $tags, null, ['class' => 'form-control']) !!}
+                <select  name="tag"
```

Comment:
You can replace the label and field (because right now it's also not using the custom module name for tags) with only `@include('cruds.fields.tag', ['model' => $tag, 'allowNew' => false])` :-)  And replace the field name from `tag` to `tag_id` to keep the code consistant (change in service and request validator)